### PR TITLE
[MUGEN][Doc] Add common README

### PR DESCRIPTION
**Pull request by langong347**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #356
* __->__ #357
* #355
* #354

Summary:
- Add a top level README for common stuff between the retrieval and
  generation sub-examples
- Remove duplicated content from the lower level READMEs

Test Plan:
Verified the content and links.

Differential Revision: [D40421597](https://our.internmc.facebook.com/intern/diff/D40421597)

### CODE REVIEW

**Comment by langong347** [comment]
@langong347 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40421597).

**Comment by langong347** [comment]
@langong347 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40421597).

File: examples/mugen/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# MUGEN: A Playground for Video-Audio-Text Multimodal Understanding and GENeration
+
+Implementation of [MUGEN](https://arxiv.org/abs/2204.08058) in TorchMultimodal for text-video [retrieval](https://github.com/facebookresearch/multimodal/tree/main/examples/mugen/retrieval) and [generation](https://github.com/facebookresearch/multimodal/tree/main/examples/mugen/generation).
```
Comment (ebsmothers) [inline examples/mugen/README.md:3]: nit: would make it explicit here that more detailed instructions for each of the two tasks can be found in the linked directories.

**Review by ebsmothers — APPROVED** [review]

**Comment by langong347** [comment]
@langong347 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40421597).

**Pull request closed without merging**